newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrObjectSelector.js

```diff
@@ -272,16 +298,40 @@ export default class InstructionOrObjectSelector extends React.PureComponent<
       onSearchStartOrReset,
       onClickMore,
     } = this.props;
+
+    const objectsContainersList = projectScopedContainersAccessor
+      .get()
+      .getObjectsContainersList();
+
+    if (objectsContainersList.getObjectsContainersCount() === 0) {
+      throw new Error(
+        'Called enumerateObjectsAndGroups without any object container.'
+      );
+    }
+    // TODO Use a loop instead of looking for 2 object containers.
+    if (objectsContainersList.getObjectsContainersCount() > 2) {
+      console.error(
+        'Called enumerateObjectsAndGroups with more than 2 object containers.'
+      );
+    }
```

Comment:
```suggestion
    if (objectsContainersList.getObjectsContainersCount() === 0) {
      throw new Error(
        'Used InstructionOrObjectSelector without any object container.'
      );
    }
    // TODO Use a loop instead of looking for 2 object containers.
    if (objectsContainersList.getObjectsContainersCount() > 2) {
      console.error(
        'Used InstructionOrObjectSelector with more than 2 object containers.'
      );
    }
```